tools/resourcedocsgen/pkg/docs/templates/examples.tmpl

```diff
@@ -10,7 +10,7 @@
 
 {{ range $key, $val := .Snippets }}
 {{ htmlSafe "<div>" }}
-{{ htmlSafe "<pulumi-choosable type=\"language\" values=\"" }}{{ $key }}{{ htmlSafe "\">" }}
+{{ htmlSafe "<pulumi-choosable type=\"language\" values=\"" }}{{ if eq $key lNodejs }}typescript{{ else }}{{ $key }}{{end}}{{ htmlSafe "\">" }}
```

Comment:
I think this is also needed for all resources and functions

Reply:
I don't see any examples for functions, only type signatures (which work for resources also).

Reply:
Ah yeah, you're right. I was apparently looking at an old preview of the deployed docs site which didn't have TS docs showing for resources.